roadmap.md

```diff
@@ -0,0 +1,187 @@
+# Tekton Mission and Roadmap
+
+This doc describes Tekton's mission and the 2020 roadmap primarily
+for Tekton Pipelines but some sneak peeks at other projects as well.
+
+- [Mission and Vision](#mission-and-vision)
+- [Roadmap](#roadmap)
+  - [Beta and GA releases](#beta-and-ga)
+  - ["Feature Complete" Tekton](#feature-complete-tekton)
+    - [Task Interfaces and PipelineResources](#task-interfaces-and-pipelineresources)
+  - [Triggers](#triggers)
+  - [SCM support](#scm-support)
+  - [Catalog](#catalog)
+  - [Results](#results)
+  - [Release and dogfooding](#release-and-dogfooding)
+
+## Mission and Vision
+
+Tekton's mission:
+
+  Be the industry-standard, cloud-native CI/CD platform and ecosystem.
```

Comment:
> I wonder if this is misleading or not : does tekton aims to be a CI/CD platform on its own or does it aim to provide all the component necessarly to have a CI/CD platform ? (the different, sublte, being that tekton stays un-opiniated)

How about "platform components" instead of "platform"?

> This is something that can be confusing to end user : do I want to go with straight up Tekton or for something that integrate tekton ?

Definitely already confusing! I'm not even sure the answer is clear at the moment. I think that's what I'm trying to get at in the section with the 2 groups of engineers, if you're building a CI/CD system, use Tekton, if not, use something that supports Tekton